src/Setup/DevDivVsix/CompilersPackage/Microsoft.CodeAnalysis.Compilers.swr

```diff
@@ -8,10 +8,10 @@ vs.dependencies
   vs.dependency id=Microsoft.Net.4.6.1.FullRedist.Resources
 
 folder InstallDir:\MSBuild\15.0\Bin
-    file source=$(OutputPath)\VBCSCompiler.exe vs.file.ngen=yes
```

Comment:
I wonder if we could do a search on other .swr files in the VS repo and see if we can't find other people making the same mistake?